src/formatDate.js

```diff
@@ -8,7 +8,42 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separator = fromFormat.pop();
+  const newSeparator = toFormat.pop();
```

Comment:
Similarly, using `Array.pop()` on the `toFormat` array is also mutating the input, which should be avoided.